show-actual-battery-percent

//show actual battery percent in the system user interface.
//note: battery @ 100% will only charge on a new charge event or if power is <94%
Index: /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
===================================================================
--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -1168,11 +1168,11 @@ handlePowerNotifications: function(paylo
 	
 	Mojo.Log.info("SystemUI Power Payload "+ Object.toJSON(payload));	
 	// Is the battery level provided?
-	if (payload.percent_ui != undefined) {	
+	if (payload.percent != undefined) {	
 		//Save the Battery Percentage
-		this.batteryLevel = payload.percent_ui;	
+		this.batteryLevel = payload.percent;	
 		for (var i = 0; i < this.powerIndicator.length; i++) {
-			if (payload.percent_ui > this.powerIndicator[i]) 
+			if (payload.percent > this.powerIndicator[i]) 
 				continue;
 			this.lastBatteryLevel = i;
 			this.lastBatteryLevelPayload = payload;
@@ -1183,7 +1183,7 @@ handlePowerNotifications: function(paylo
 				$('power').className = 'battery-' + i;
 				
 				//Show Banner Message if the Battery level is below 20%
-				var batteryalert = $L("#{percent_ui}% battery remaining").interpolate(payload);	
+				var batteryalert = $L("#{percent}% battery remaining").interpolate(payload);	
 				if(this.batteryLevel <= 5 && !this.batteryLevel5Shown) {
 					this.batteryLevel5Shown = true;
 					this.showLowBatteryAlert();
@@ -1209,7 +1209,7 @@ handlePowerNotifications: function(paylo
 			}
 			else {
 				Mojo.Log.info("SystemUI- Is Charging "+ this.isCharging );
-				if(payload.percent_ui == 100)
+				if(payload.percent == 100)
 					$('power').className = 'battery-charged';
 				else
 					$('power').className = 'battery-charging-' + i;
